### ISSUE TITLE
Polyline is not visible.

### ISSUE THREAD

**Issue by devansh12b2** [CCExtractor/beacon#31]
In map the polyline is not visible.

**Comment by nb9960** [comment]
@cfsmp3 For polyline to work, billing needs to be activated on google cloud platform. [Ref](https://www.fluttercampus.com/guide/75/how-to-solve-poly-line-not-drawing-on-google-map-flutter/#3-activate-billing-on-google-cloud-platform)

**Comment by cfsmp3** [comment]
> @cfsmp3 For polyline to work, billing needs to be activated on google cloud platform. [Ref](https://www.fluttercampus.com/guide/75/how-to-solve-poly-line-not-drawing-on-google-map-flutter/#3-activate-billing-on-google-cloud-platform)

This is specific to each backend deployment, isn't it? I think you are still using aadi's, right? 

What we should do ASAP is get an official dev backend (but I need a proper installer and/or instructions, I don't want to put a lot of fight here because that's a sign we're not ready for production, and the idea is that anyone can have their own deployments) that I can link with my credit card and be reasonably confident it's not going to be abused :-)

**Comment by nb9960** [comment]
> > @cfsmp3 For polyline to work, billing needs to be activated on google cloud platform. [Ref](https://www.fluttercampus.com/guide/75/how-to-solve-poly-line-not-drawing-on-google-map-flutter/#3-activate-billing-on-google-cloud-platform)
> 
> This is specific to each backend deployment, isn't it? I think you are still using aadi's, right?
> 
> What we should do ASAP is get an official dev backend (but I need a proper installer and/or instructions, I don't want to put a lot of fight here because that's a sign we're not ready for production, and the idea is that anyone can have their own deployments) that I can link with my credit card and be reasonably confident it's not going to be abused :-)

No, this is not related to backend. Currently, `Google Maps API Key` is being used in the frontend for maps to work and is passed as env variable but premium features of google maps (like polyline) are paid so billing needs to be done

**Comment by cfsmp3** [comment]
> No, this is not related to backend. Currently, `Google Maps API Key` is being used in the frontend for maps to work and is passed as env variable but premium features of google maps (like polyline) are paid so billing needs to be done

Where is that key stored and why what mechanism can be ensure that only our "official" frontend deployment can use it? 

Obviously I don't want to pay for everybody's use of maps :-)
